exaudfclient/base/javacontainer/script_options/converter_legacy.cc

```diff
@@ -21,6 +21,23 @@ void ConverterLegacy::convertExternalJar(const std::string& value) {
     }
 }
 
+void ConverterLegacy::convertJvmOption(const std::string & value) {
+    for (size_t start = 0, delim = 0; ; start = delim + 1) {
+        start = value.find_first_not_of(m_whitespace, start);
+        if (start == std::string::npos)
+            break;
+        delim = value.find_first_of(m_whitespace, start);
+        if (delim != std::string::npos) {
+            m_jvmOptions.push_back(value.substr(start, delim - start));
+        }
+        else {
+            m_jvmOptions.push_back(value.substr(start));
+            break;
+        }
+    }
+}
```

Comment:
```suggestion
void ConverterLegacy::convertJvmOption(const std::string& value) {
    std::string::size_type start = 0, delim = 0;
    while ((start = value.find_first_not_of(m_whitespace, delim)) != std::string::npos) {
        delim = value.find_first_of(m_whitespace, start);
        m_jvmOptions.push_back(value.substr(start, delim - start));
    }
}

```

Reply:
Hm, if there is no whitespace in value, delim at line 27 gets and std::string::npos. Then what is the expected result of std::substr if the second parameter is some negative number. I can't find that in the documentation.

Reply:
Done